tests/e2e/specs/out-of-game/error.spec.js

```diff
@@ -0,0 +1,13 @@
+describe('Error Page', () => {
```

Comment:
Added tests for it, too. These will all obviously change, but bones are there.

Reply:
Let's call this NotFoundPage
```suggestion
describe('Not Found Page', () => {
```